src/components/RecordEditor.js

```diff
@@ -211,6 +211,15 @@ class RecordEditor extends React.Component {
 		})
 	}
 
+	handleClose() {
+		if (!Object.keys(this.state.record).includes('pid')) {
```

Comment:
I think it would be better not to hard code `pid` in there and instead use the id field specified in the config file. I guess that would be something like `this.config.id_field`?